distribution/src/bin/elasticsearch

```diff
@@ -45,6 +50,15 @@ then
   fi
 fi
 
+if [[ $AUTO_CONFIG = true ]]; then
```

Comment:
I don't see anything that would be too problematic to do in `elasticsearch.bat` . Have you looked into it at all?

Reply:
I haven't looked but this is all analogous to cli tools (plus env var checks). Should not be a problem.

Reply:
++ just bringing it up as it occurred to me while reading this through